extensions/positron-python/python_files/posit/positron/tests/test_data_explorer.py

```diff
@@ -729,6 +729,14 @@ def test_pandas_get_schema(dxf: DataExplorerFixture):
             "string",
             "string",
         ),
+        (
+            np.array(
+                ["NaT", 3600000000000, -3600000000000, 0, 0],
+                dtype="timedelta64[ns]",
+            ),
+            "timedelta64[ns]",
+            "interval",
+        ),
```

Comment:
yeah perhaps another test here with a different resolution like `[s]` just in case?

Reply:
It seems that pandas 1.5.x upcasts seconds to nanoseconds! I'll see if I can work around it